book/src/SUMMARY.md

```diff
@@ -11,6 +11,7 @@
   - [Lightwalletd](user/lightwalletd.md)
   - [zk-SNARK Parameters](user/parameters.md)
   - [Mining](user/mining.md)
+  - [Mining on Testnet](user/mining-testnet-s-nomp.md)
```

Comment:
Should we nest this under 'Mining'?

```suggestion
    - [Testing with s-nomp](user/mining-testnet-s-nomp.md)
```

Reply:
I'd also like us to clearly mark it as "not production", because we turned off a bunch of functionality that is usually used in production mining pools:

```suggestion
    - [Testnet Mining with s-nomp](user/mining-testnet-s-nomp.md)
```

Reply:
Thanks, done!

Reply:
I thin the title changed but we don;t marked more clear that the guide is not production. I made a ne[w suggestion](https://github.com/ZcashFoundation/zebra/pull/6220#pullrequestreview-1316016813) for that.